README.md

```diff
@@ -59,3 +59,34 @@ IMPORTANT! The `importer` crate's dependency on `osm2streets` must be kept in
 sync with the git version used by ATIP's [route info
 crate](https://github.com/acteng/atip/tree/map_model/route_info). Otherwise,
 the binary file format may be incompatible. Use `cargo update -p osm2streets`.
+
+## POIs (points of interest)
+
+ATIP loads extra layers showing key trip generators. These layers are
+England-wide, rather than being split into a file per area, because they're
+being used on the country-wide scheme browse page. Each layer is a single
+[PMTiles](https://protomaps.com/docs/pmtiles/) file.
+
+To run this:
+
+1.  Get `england-latest.osm.pbf` from Geofabrik. The `split_uk_osm.sh` script above does this.
+2.  Install [osmium](https://osmcode.org/osmium-tool)
+3.  Install [tippecanoe](https://github.com/felt/tippecanoe) for transforming GeoJSON to PMTiles
```

Comment:
Note another dependency here. I had no trouble installing it locally. Right now I'm the only one running things in this reop, but happy to work on ensuring there's a Docker image that can do everything too.